src/recovery/cubic.rs

```diff
@@ -230,31 +230,35 @@ fn on_packet_acked(
         r.bytes_acked_sl += packet.size;
 
         if r.bytes_acked_sl >= r.max_datagram_size {
-            r.congestion_window += r.max_datagram_size;
+            r.congestion_window += if r.hystart.in_css(epoch) {
```

Comment:
nit: I find this code construction a bit hard to grok. Could maybe split the conditional into out

Reply:
updated.